hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -36,16 +37,16 @@
   private Server server;
   private Engine engine;
   private HdfsFrameworkConfig hdfsFrameworkConfig;
-  private PersistentState persistentState;
```

Comment:
Same comment on the Impl of the interface. Also, how do you feel about making it IPersistentStateStore since it is an interface and using that naming convention?